docs/source/application_developer_guide.rst

```diff
@@ -196,7 +196,10 @@ decorator, so they will only be compiled when compiling into Wasm.
       let handler = IntkeyMultiplyTransactionHandler::new();
       match handler.apply(request, context) {
           Ok(_) => Ok(true),
-          Err(err) => Err(err)
+          Err(err) => {
+              info!("{}", err);
```

Comment:
should this be `error`?

Reply:
Info is the level that the tp processor prints out the log saying why it was invalid. So i kept it the same at this level.